### PR TITLE
feat(profiling): add recent profiles tab to summary list

### PR DESCRIPTION
**Pull request by JonasBa**

### CODE REVIEW

File: static/app/components/profiling/profileEventsTable.tsx [hunk 1]
```diff
@@ -154,6 +154,34 @@ function ProfileEventsCell<F extends FieldType>(props: ProfileEventsCellProps<F>
...
 
+  if (key === 'trace') {
+    const traceId = props.dataRow[key];
+    if (!traceId) {
+      return <Container>{t('n/a')}</Container>;
+    }
+
+    return (
+      <Container>
+        <Link to={`/performance/trace/${traceId}`}>{traceId}</Link>
```
Comment (Zylphrex) [inline static/app/components/profiling/profileEventsTable.tsx:165]: Let's format all ids with `getShortEventId`

File: static/app/views/profiling/profileSummary/recentProfilesTable.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+type FieldType = (typeof ALL_FIELDS)[number];
+
+export function RecentProfilesTable() {
+  const location = useLocation();
+  const organization = useOrganization();
+
+  const profilingUsingTransactions = organization.features.includes(
+    'profiling-using-transactions'
+  );
+  const fields = profilingUsingTransactions ? ALL_FIELDS : BASE_FIELDS;
```
Comment (Zylphrex) [inline static/app/views/profiling/profileSummary/recentProfilesTable.tsx:37]: I would not include `user_misery()` here as it's an aggregation. This table should just be the raw fields.

**Review by Zylphrex — COMMENTED** [review]
Thoughts on renaming this tab `Profiles`? You should be able to sort the table on any of the sortable columns defined already so it can show more than just recent profiles.

**Comment by JonasBa** [comment]
@Zylphrex agree. Let me rename it

File: static/app/views/profiling/profileSummary/profilesTable.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  'trace.transaction',
+  'profile.duration',
+  'timestamp',
+  'release',
+  'environment',
+  'os.name',
+  'os.version',
+] as const;
+
+// user misery is only available with the profiling-using-transactions feature
```
Comment (Zylphrex) [inline static/app/views/profiling/profileSummary/profilesTable.tsx:25]: outdated comment?

**Review by Zylphrex — APPROVED** [review]

File: static/app/views/profiling/profileSummary/profilesTable.tsx (hunk 3 above)
Comment (JonasBa) [reply static/app/views/profiling/profileSummary/profilesTable.tsx]: Removed!

**Pull request merged**